cms/settings/base.py

```diff
@@ -968,9 +968,12 @@
 
 # Configuration for the External Search service
 SEARCH_INDEX_PUBLISHER_BACKEND = os.getenv("SEARCH_INDEX_PUBLISHER_BACKEND")
+# TODO: Should we default to "kafka" or leave it empty?
+# TODO: No need for empty string as KafkaProducer will default to localhost:9092
 KAFKA_SERVERS = os.getenv("KAFKA_SERVERS", "").split(",")
 KAFKA_USE_IAM_AUTH = os.getenv("KAFKA_USE_IAM_AUTH", "false").lower() == "true"
-KAFKA_API_VERSION = tuple(map(int, os.getenv("KAFKA_API_VERSION", "3,5,1").split(",")))
+# TODO: We can use the dot notation here
```

Comment:
nitpick: no longer valid.

Can you sort out / clean up these TODOs?

Reply:
Done! [a6748f8](https://github.com/ONSdigital/dis-wagtail/pull/300/commits/a6748f851f5021db2f0b3bc483cf942e2ddbaf24) and [4ec1288](https://github.com/ONSdigital/dis-wagtail/pull/300/commits/4ec12888c8894efd00aa62b683a46996218e4414)